server/channels/app/teams/helper_test.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/mattermost/mattermost/server/public/shared/request"
 	"github.com/mattermost/mattermost/server/v8/channels/store"
 	"github.com/mattermost/mattermost/server/v8/config"
+	"github.com/stretchr/testify/require"
```

Comment:
Nit: Please move this import into a separate group before the `github.com/mattermost/mattermost/server`  ones